### ISSUE TITLE
Custom Components Share Space

### ISSUE THREAD

**Issue by KipOmaha** [KipOmaha/Dashboard#24]
@doriansmiley, was wondering if you could add wisdom to this issue. concerning making it compatible with your component sharing application for custom elements.

Sort of a best strategy to get a dashboard made for custom elements sharing and editing to be in sync with an indesign server type of setting.

**Comment by KipOmaha** [comment]
Going to push a directory to share the demos of the x-tag components that I'm updating.

It's going to be like a dashboard, but with a different layout then the current one on the GH-Page.  I'm going to push the 'x-tabbox' dashboard as the, `Open Components Dashboard Space`.

@csuwildcat, @doriansmiley this is something that I've been wanting to speak to you both about whenever you have time. 

The x-components part of my project for me maybe coming to a completion for now, but I still want to know about putting the parts that I'm building or am going to build for my Online Hobby Shop to a components shop or space of some sort so that I could advertise the online hobby shop project and share the software with others. So I was wondering if either @csuwildcat or @doriansmiley have projects regarding this?  

I don't know if that is ever going to be a thing that either of you are going to do, I have read chatter on threads someplace about it before but haven't really seen one. 

I want to do this for a couple of reasons.

1. Online custom shops in the recent years are popping up, from Teddy bears, clothes, to custom computers. All these web apps help people design it themselves and generates interest from the public regarding peoples creations.
-- With all the competition a group of collaborators each running a small business custom shop would find it a lot easier to share software rather than each one re-inventing the wheel.
-- Since custom shops are by nature each unique a group of people would find it more useful to create ad networks to advertise each others work. This is beneficial for customers and shop owners because it will create familiarity with your UI you have at your shop versus competitors who don't use the same "web brand components". [BRAND RECOGNITION IS IMPORTANT] 

2. Sharing web software created for specific professional purposes makes sense [although often times it doesn't happen], because it helps create an ecosystem around your particular area of business. It's an instance where quantity outweighs quality [I say this lightly], if people get used to doing something they will continue to go out looking for those things, because that's just what they know.
-- So basically it would help create more buzz for the type of custom online store.
-- A Search engine could be optimized across component applications that all share the same meta schema.
3. Economic factors in today's society have people looking for self employment as a part time job, technology like this can be affordable and easy to access.
-- With technology like this people could sell custom wares all over the world. It expands the market for small businesses.

**Comment by KipOmaha** [comment]
Sorry for the essay
